habitat-lab/habitat/articulated_agent_controllers/humanoid_base_controller.py

```diff
@@ -29,7 +30,7 @@ class Motion:
         :param joints_quat_array: num_poses x num_joints x 4 array, containing the join orientations
         :param transform_array: num_poses x 4 x 4 array, containing the root transform
         :param displacement: on each pose, how much forward displacement was there?
-            Used to measure how many poses we should advance to move a cerain amount
+        Used to measure how many poses we should advance to move a cerain amount
```

Comment:
I.e., this will most probably cause a problem -- it should be indented like before, or on a single line.